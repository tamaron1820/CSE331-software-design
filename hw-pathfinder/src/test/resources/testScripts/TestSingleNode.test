# Test whether Dijkstra's algorithm works correctly
# if there is a single node which has no edges

CreateGraph graph1

AddNode graph1 n1

FindPath graph1 n1 n1