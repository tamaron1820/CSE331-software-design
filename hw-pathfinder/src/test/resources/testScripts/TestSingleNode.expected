# Test whether Dijkstra's algorithm works correctly
# if there is a single node which has no edges

created graph graph1

added node n1 to graph1

path from n1 to n1:
total cost: 0.000
